generated/nidmm/session.py

```diff
@@ -369,7 +369,7 @@ class _SessionBase(object):
     Specifies the number of measurements the DMM takes each time it receives a  trigger in a multiple point acquisition.
     '''
     sample_interval = _attributes.AttributeViReal64TimeDeltaSeconds(1250303)
-    '''Type: datetime.timedelta
+    '''Type: float
```

Comment:
Inconsistent, doesn't mention datetime.timedelta anymore.
There are others with the same problem.

Reply:
Another place needed to use `'type_in_documentation'`. Fixed